File: sample-kanban-jvm/src/main/kotlin/sample/testcontainer/kanban/models/Status.kt
package sample.testcontainer.kanban.models

import jakarta.persistence.*
import java.time.LocalDateTime

@Entity
@Table(name = "status", schema = "kanban")
class Status(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,
    var description: String? = null,
    var meansComplete: Boolean? = null,
    @Column(updatable = false)
    @Temporal(TemporalType.TIMESTAMP)
    var created: LocalDateTime? = null
)

File: sample-kanban-go/app/models/status.go
package models

import "time"

type Status struct {
	Id            int64      `db:"id" goqu:"skipinsert"`
	Description   string     `db:"description"`
	MeansComplete bool       `db:"means_complete"`
	Created       *time.Time `db:"created" goqu:"omitnil"`
}
